cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -2581,6 +2636,23 @@ pub fn define(
         .operands_out(vec![a]),
     );
 
+    let a = &operand("a", b1);
+    let x = &operand("x", Ref);
+
+    ig.push(
+        Inst::new(
+            "isnull",
```

Comment:
light preference to name this `is_null`, as we have other instances of names like that ("load_complex", etc.).

Reply:
Hm, @sunfishcode made a commit a while ago specifically to rename the instruction from `is_null` to `isnull`. (https://github.com/CraneStation/cranelift/pull/781/commits/66489aeb5bf2f964f95ece0be7988ca4ac65e5be)

It is probably because it shadowed the other method we have in our codebase (which led to the bad search-and-replace). I think the use of the underscore is inconsistent within Cranelift IR since we also have instructions like `trapif`, `extractlane`, and `debugtrap`. I don't mind changing it back to `is_null`, but after maybe @sunfishcode tell us why he chose to exclude the underscore?

Reply:
I don't recall a strong reason. I'm ok adding the underscore back.